analysis_options.yaml

```diff
@@ -2,17 +2,92 @@ include: package:pedantic/analysis_options.yaml
 analyzer:
   strong-mode:
     implicit-casts: false
-  errors:
-    todo: ignore
```

Comment:
Why remove this?

Reply:
Much noise. We're running travis w/ `--fatal-hints`

Reply:
This doesn't add any noise... It doesn't impact travis at all.

Reply:
Noise to this file. Why these? Why not others? We fail CI for any items, so flagging some as errors vs others doesn't add much – AFAIC